c++/src/kj/compat/http-test.c++

```diff
@@ -5691,7 +5691,11 @@ KJ_TEST("CONNECT Server cancels read w/client") {
 
     bool failed = false;
     io->write("hello", 5).catch_([&failed](kj::Exception&& ex) {
+#if KJ_HTTP_TEST_USE_OS_PIPE
+      KJ_ASSERT(ex.getDescription() == "::write(fd, buffer, size): Broken pipe");
```

Comment:
Perhaps it would be best to check for a DISCONNECTED exception and not rely on specific message text.

Reply:
Done. Also squashed out the fixup commit.